src/mmrelay/migrate.py

```
@@ -0,0 +1,2588 @@
+        old_community_dir = old_plugins_dir / "community"
+        if _migrate_plugin_tier(
+            old_community_dir,
+            staging_dir / "community",
+            "community",
+            force,
+            errors,
+        ):
+            migrated_types.append("community")
+
+        if errors:
+            raise OSError("; ".join(errors))
```

Ruff (0.14.14)

[warning] Abstract `raise` to an inner function

(TRY301)